example/lib/home/home.dart

```diff
@@ -72,11 +73,19 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future<void> _getSystemDevices() async {
+    // For macOS and iOS, it is recommended to set a filter to get system devices
+    if (defaultTargetPlatform == TargetPlatform.macOS ||
+        defaultTargetPlatform == TargetPlatform.iOS &&
+            (scanFilter?.withServices ?? []).isEmpty) {
+      showSnackbar(
+          "No services filter was set for getting system connected devices");
```

Comment:
```suggestion
          "No services filter was set for getting system connected devices. Using default...");
```

Reply:
@rohitsangwan01 I think we should print a log for whoever calls this API without the parameter, not just in the example app.